tests/factories/cht/settings/privacy-policy.js

```diff
@@ -0,0 +1,38 @@
+const Factory = require('rosie').Factory;
+const privacyPolicyHtml = ({ header, paragraph }) => {
+  return `
+<div>
+  <h1>${header}</h1>
+  <p>${paragraph}</p>
+</div>
+`;
+};
+
+const english = { header: 'English Privacy Policy', paragraph: 'More markup' };
+const french = { header: 'Politique de confidentialité en Francais', paragraph: 'Plus de markup' };
+const privacyPolicyInFrench = privacyPolicyHtml(french);
+const privacyPolicyInEnglish = privacyPolicyHtml(english);
+
+
+
+const privacyPolicy = () => {
+  return new Factory()
```

Comment:
I'm interested in the purpose of using rosie for defining this document.

Reply:
The privacy policy represents a document object in the database. Since we can define them once and close to real data I moved it to a factory object.